docker/pypi/dmwm-base/monitor.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# start process exporter
+configs="config config-monitor config-output config-transferor config-ruleCleaner config-unmerged"
+for p in $configs; do
+    pat="wmc-httpd.*$p"
+    pid=`ps axjfwww | grep "$pat" | grep -v grep | grep -v process_monitor | grep -v " 1 " | awk '{print $1}'`
+    if [ -n "$pid" ]; then
+        app=`grep ^main.application /etc/secrets/${p}.py | sed -e 's,#.*,,g' | awk '{split($0,a,"="); print a[2]}' | sed -e "s, ,,g" -e 's,",,g'`
+        prefix=process_exporter_${app}
+        port=`grep main.port /etc/secrets/config.py | sed -e 's,#.*,,g' | awk '{split($0,a,"="); print a[2]}' | sed -e "s, ,,g"`
```

Comment:
Maybe you wanted to write `/etc/secrets/${p}.py`?

Reply:
good catch, I pushed the changes.

Reply:
and, I rebuilt and uploaded again dmwm-base image to registry with the same tag